content/blog/devmode-vs-prodmode.md

```diff
@@ -0,0 +1,31 @@
+---
+date: 2024-06-30
+title: Production Mode vs. Dev Mode
+---
+
+When setting up workflow orchestration, it's crucial to understand the differences between production mode and development (dev) mode, particularly in terms of infrastructure requirements. This distinction ensures that workflows are efficiently managed and executed based on their intended use case. Here, we'll explore these differences, focusing on the infrastructure required to run workflows in each mode.
+
+# Production Mode
+Production mode is tailored for environments where stability, reliability, and scalability are paramount. The Orchestrator Helm chart or Orchestrator operator is designed specifically to meet the demanding requirements of production environments. Key requirements include:
```

Comment:
Links to both?

Reply:
I can only include a link to the production mode which is the only installation method described in this website.
I'd be happy if someone with experience with dev-mode can create a blog to explain how to work with it and then add a reference to it from here.

Reply:
Agree that the dev mode isn't explained.
Created a Jira story to track this [FLPATH-1451](https://issues.redhat.com/browse/FLPATH-1451)